packages/api/package.json

```diff
@@ -21,7 +21,8 @@
     "check-format": "prettier --check src",
     "eslint": "eslint --max-warnings 0 ./src --ext .ts",
     "lint-sql": "docker run -i --rm -v $PWD:/sql sqlfluff/sqlfluff:2.3.5 lint --dialect postgres --nocolor src/*/queries/ src/fixtures",
-    "lint": "npm run check-format && npm run check-types && npm run eslint && npm run lint-sql",
+    "lint-docs": "redocly lint docs/future/api.yaml",
+    "lint": "npm run check-format && npm run check-types && npm run eslint && npm run lint-sql && npm run lint-docs",
```

Comment:
This is great!

In some projects I've seen the pattern of naming sub-lint scripts as `lint:foo` (e.g. `lint:format`, `lint:types`, `lint:eslint`, `lint:sql`) and then having the scripts in alphabetical order.  This is an incredibly minor note, and I know the pattern is already established, but just to put in your ear for future in case it sounds appealing!